src/adapters/rubicon.js

```diff
@@ -167,7 +179,7 @@ function RubiconAdapter() {
 
     ads.forEach(function (ad) {
       if(ad.status !== 'ok') {
-        throw 'bad ad status';
+        throw new RubiconError('bad ad status');
       }
 
       //store bid response
```

Comment:
any exceptions generated below here will not result in a "badbid" being added to the bidmanager.  I think once we're processing a bid we should make sure to always add it to the bidmanager, as either a good or bad bid.  So we should handle any exceptions not generated from `bidmanager.addBidResponse()` below.